src/main/java/appeng/tile/inventory/AppEngItemFilters.java

```diff
@@ -0,0 +1,43 @@
+
+package appeng.tile.inventory;
+
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+
+public class AppEngItemFilters
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1118)